src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -693,7 +704,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             BlobCacheAttribute.AddService("default", privateCacheService);
             BlobCacheAttribute.AddService("public", publicCacheService);
 
-            UpdateDatabase(app, env);
+            if (_applyDatabaseMigrations)
```

Comment:
Maybe something like `isTestStartup`?

Alternatively, we could do some more messing around with `HostEnvironment` to setup a test environment instead of piggyback on Development. This would be a bit more involved and might mean duplicating config from `appsettings.Development.json` into `appsettings.IntegrationTests.json` or `appsettings.Test.json` (or similar).

Reply:
Actually I really like the environment idea, so I'm going to have a look into that, ta

Reply:
Hey @ntsim - I've made this change but as it's not trivial, I've done it as a separate PR back into this branch.  Please take a look when you have time - https://github.com/dfe-analytical-services/explore-education-statistics/pull/4317.  Ta